### DISCUSSION TITLE
Install 2.3 or wait for 2.4

### DISCUSSION THREAD

**Discussion by Bunrs21** [Security-Onion-Solutions/securityonion#10851] [Unsupported Versions] [answered]
Hello,

I started to deploy Security Onion and while looking up the instructions on how to integrate Wazuh I found an article stating that 2.4 is coming out this year and will be removing Wazuh and replacing it. So, my question is it worth my time continuing my 2.3 deployment now or should I wait until the 2.4 release?

Alternatively, is there a way to continue with 2.3 and just use the new Elastic Agent instead?

Thank you for your assistance.

**Comment by dougburks** [comment, marked as answer]
I would just wait for 2.4.

2.4 Release Candidate 1 is coming soon!

**Reply by Bunrs21** [reply]
Thank you for your response. I'll wait.
